xcube_sh/chunkstore.py

```diff
@@ -151,6 +151,8 @@ def __init__(self,
         self._add_static_array('time', t_array, time_attrs)
         self._add_static_array('time_bnds', t_bnds_array, time_bnds_attrs)
 
+
```

Comment:
Why so many blank lines? I think one was enough.

Reply:
PEP 8 agrees with you ("Use blank lines in functions, sparingly...").

Reply:
It should be 1 line only.